forte/data/data_store.py

```diff
@@ -853,15 +883,18 @@ def get(
                 if issubclass(get_class(type), entry_class):
                     all_types.append(type)
             for type in all_types:
-                for entry in self.__elements[type]:
-                    yield entry
+                yield from self.skip_none_iterator(self.__elements[type])
         else:
             try:
                 entries = self.__elements[type_name]
             except KeyError as e:
                 raise KeyError(f"type {type_name} does not exist") from e
-            for entry in entries:
-                yield entry
+            yield from self.skip_none_iterator(entries)
+
+    def skip_none_iterator(self, entry: list) -> Iterator[List]:
```

Comment:
don't need to call this `skip_none_iterator`, because this name exposes the internal implementation, just say this is the iterator to use. It can be as simple as `iter()`, also the input shouldn't be the entry list (the actual data), it should be a `type_name`.